Week1/exercise1.js

```diff
@@ -0,0 +1,139 @@
+const mysql = require("mysql");
+const connection = mysql.createConnection({
+  host: "localhost",
+  user: "hyfuser",
+  password: "hyfpassword",
+});
+
+connection.connect((error) => {
+  if (error) {
+    throw error;
+  }
+  console.log("MySql connected...");
+});
+
+connection.query("CREATE DATABASE IF NOT EXISTS meetup", (error) => {
+  if (error) {
+    console.error("Error creating database: ", error);
+  } else {
+    console.log("Database created");
+  }
+});
+
+// Switch to the meetup database
+connection.query("USE meetup", (error) => {
+  if (error) {
+    console.error("Error switching to database: ", error);
+  } else {
+    console.log("Using database: meetup");
+  }
+});
+
+connection.query('DROP TABLE invitee');
+connection.query('DROP TABLE meeting');
+connection.query('DROP TABLE room');
```

Comment:
Dropping a table without considering it's existence condition while creating the database will result in an error, to make sure we won't have the fatal error `ERROR 1051 (42S02): Unknown table 'meetup.invitee'` you might want to use the `if exists` condition:
```suggestion
connection.query("DROP TABLE IF EXISTS invitee");
connection.query("DROP TABLE IF EXISTS meeting");
connection.query("DROP TABLE IF EXISTS room");
```

And that means if the table didn't exist before you dropping it, your code won't run.